llvm-general/test/LLVM/General/Test/Metadata.hs

```diff
@@ -16,45 +16,45 @@ import qualified LLVM.General.AST.Constant as C
 import LLVM.General.AST.Global as G
 
 tests = testGroup "Metadata" [
-  testCase "local" $ do
-    let ast = Module "<string>" Nothing Nothing [
-          GlobalDefinition $ globalVariableDefaults { G.name = UnName 0, G.type' = i32 },
-          GlobalDefinition $ functionDefaults {
-            G.returnType = i32,
-            G.name = Name "foo",
-            G.basicBlocks = [
-              BasicBlock (UnName 0) [
-                 UnName 1 := Load {
-                            volatile = False,
-                            address = ConstantOperand (C.GlobalReference (ptr i32) (UnName 0)),
-                            maybeAtomicity = Nothing,
-                            A.alignment = 0,
-                            metadata = []
-                          }
-                 ] (
-                 Do $ Ret (Just (ConstantOperand (C.Int 32 0))) [
-                   (
-                     "my-metadatum", 
-                     MetadataNode [
-                      Just $ LocalReference i32 (UnName 1),
-                      Just $ MetadataStringOperand "super hyper",
-                      Nothing
-                     ]
-                   )
-                 ]
-               )
-             ]
-           }
-         ]
-    let s = "; ModuleID = '<string>'\n\
-            \\n\
-            \@0 = external global i32\n\
-            \\n\
-            \define i32 @foo() {\n\
-            \  %1 = load i32* @0\n\
-            \  ret i32 0, !my-metadatum !{i32 %1, metadata !\"super hyper\", null}\n\
-            \}\n"
-    strCheck ast s,
+  -- testCase "local" $ do
```

Comment:
I assume you've commented these out due to build issues? If not could you uncomment them? I'd like to help get this pr :green_apple:!